pkg/cache/redis.go

```diff
@@ -67,3 +65,43 @@ func (c *redisCache) SetAccessList(ctx context.Context, accessList rbac.AccessLi
 	c.client.Set(ctx, authKey(ctx), string(buf), config.Get().Clients.Redis.Expiration)
 	return nil
 }
+
+func (c *redisCache) GetPulpContentPath(ctx context.Context) (string, error) {
+	var repoConfigFile string
+	key := pulpContentPathKey()
+	buf, err := c.get(ctx, key)
+	if err != nil {
+		return "", fmt.Errorf("redis get error: %w", err)
+	}
+
+	err = json.Unmarshal(buf, &repoConfigFile)
+	if err != nil {
+		return "", fmt.Errorf("redis unmarshal error: %w", err)
+	}
+	return repoConfigFile, nil
+}
+
+func (c *redisCache) SetPulpContentPath(ctx context.Context, repoConfigFile string) error {
+	buf, err := json.Marshal(repoConfigFile)
+	if err != nil {
+		return fmt.Errorf("unable to marshal for Redis cache: %w", err)
+	}
+
+	c.client.Set(ctx, pulpContentPathKey(), string(buf), config.Get().Clients.Redis.Expiration)
```

Comment:
i imagine we could do different expirations for different items.  I think rbac has an expiration of 1 minute today, this could probably be even longer (an hour or so).  Its not a huge deal and something we could address later, so i'm 50/50 on it

Reply:
yep, that makes sense!